.dependabot/config.yml

```diff
@@ -1,29 +1,13 @@
 # Reference: https://dependabot.com/docs/config-file/
 version: 1
 update_configs:
-  # This configures dependency updates for one package manager. In some
```

Comment:
These comments didn't seem useful, and were somewhat confusing for external readers.